docs/Pflichtenheft.tex

```diff
@@ -292,21 +294,21 @@ \subsection{Überwachungskonsole}
  \item[NF120] Ports dürfen Werte im Bereich 1024 bis 61000 annehmen.
  \item[NF130] Die Schwellenwerte für die Alarme dürfen zwischen 0 und 100 \% gewählt werden.
  \item[NF140] Die Aktualisierungsfrequenz kann im Bereich zwischen 20 und 4000 Millisekunden gewählt werden.
- \item[NF150] Wenn sich Werte in der Fertigungssimulation ändern, werden die Änderungen spätestens nach der Zeitspanne von
-   einer halben Sekunde plus dem Aktualisierungsintervall in der Überwachungskonsole angezeigt.
+ \item[NF150] Wenn sich Werte in der \gls{Fertigungssimulation} ändern, werden die Änderungen spätestens nach der Zeitspanne von
+   einer halben Sekunde plus dem Aktualisierungsintervall in der \gls{Uberwachungskonsole} angezeigt.
 \end{enumerate}
 
 \pagebreak
 \section{Produktdaten}
 \begin{enumerate}
- \item[D10] Die \gls{Uberwachungskonsole} speichert den zuletzt eingestellten Port des OPC UA Client.
- \item[D20] Die \gls{Uberwachungskonsole} speichert die zuletzt eingestellte IP-Adresse und Ports der OPC UA Server.
- \item[D30] Die \gls{Uberwachungskonsole} speichert die zuletzt eingestellte Aktualisierungsfrequenz für die Sensorwerte.
+ \item[D10] Die \gls{Uberwachungskonsole} speichert den zuletzt eingestellten Port des \gls{OPC UA} Clients.
```

Comment:
Stimmt.

Dann wird das noch ergaenzt. Um 11.15 Uhr bin ich in der Fachschaft, dann mach ich das. Um kurz vor 12 vermute ich kannst du dann wieder drueber gucken ;)